_data/projects/JavaScript.yml

```diff
@@ -0,0 +1,12 @@
+name: JavaScript
```

Comment:
@shiftkey @kshitizsaini113 Wouldn't it be better to use some other name than just JavaScript? I don't want users to get Confused with the original JavaScript programming language repository.

Reply:
Great point @ritwik12. 

@kshitizsaini113 is there a better name here to describe the project?

Reply:
I have changed the name

Reply:
Thanks